Makefile

```diff
@@ -1054,18 +1048,15 @@ compile-%: dist/Makefile.tmpl dist/configure dist/%/Makefile.basic | copy-krmlli
 # C tests (from F* files) #
 ###########################
 
-ifeq ($(OS),Windows_NT)
-OPENSSL_HOME	:= $(shell cygpath -u $(OPENSSL_HOME))
-LDFLAGS		+= -lbcrypt
-endif
-LDFLAGS 	+= -L$(OPENSSL_HOME)
-
 CFLAGS += -Wall -Wextra -g \
   -Wno-int-conversion -Wno-unused-parameter \
-  -I$(OPENSSL_HOME)/include \
-  -O3 -march=native -mtune=native -I$(KRML_HOME)/krmllib/dist/minimal \
+  -O3 -I$(KRML_HOME)/krmllib/dist/minimal \
   -I$(KRML_HOME)/include -Idist/gcc-compatible
 
+ifneq ($(shell uname -m),arm64)
```

Comment:
Why is it needed to prevent the use of these flags on arm?

Reply:
they are not recognized on arm, the compiler will error out